science/gnuradio/Portfile

```diff
@@ -1,458 +1,317 @@
     # temporary patch to fix and allow external setting for
     # CMAKE_C/CXX_STANDARD
-
     patchfiles-append \
-        patch-C_CXX_STANDARD.release.diff
-
+        patch-C_CXX_STANDARD.37.diff
     # temporary patch to fix use of GNUCXX to instead be for both GNU
     # and Clang compilers
-
     patchfiles-append \
-        patch-fix-GNUCXX.release.diff
-
+        patch-fix-GNUCXX.37.diff
     # temporary patch to fix installing CTRLPORT Python apps location
-
     patchfiles-append \
-        patch-fix-ctrlport-python-app-install-location.release.diff
-
+        patch-fix-ctrlport-python-app-install-location.37.diff
     # patch to fix using Boost 1.70.0 API
-
     patchfiles-append \
-        patch-fixes_for-boost_1.70.0.release.diff
-
+        patch-fixes_for-boost_1.70.0.37.diff
     # temporary patch to fix using Thrift 0.9.2 through 0.12.0
-
     patchfiles-append \
-        patch-fixes_for-thrift.release.diff
-
-} elseif {${subport} eq "gnuradio-devel"} {
-
-    # patches for devel
+        patch-fixes_for-thrift.37.diff
 
-    # force expansion of CMake-based template files
-
-    patchfiles-append \
-        patch-cmake-expand.devel.diff
-
-    # patch to fix CTRLPORT testing
-
-    patchfiles-append \
-        patch-fix_ctrlport_test.devel.diff
-
-    # patch to fix and allow external setting for
-    # CMAKE_C/CXX_STANDARD
-
-    patchfiles-append \
-        patch-C_CXX_STANDARD.devel.diff
-
-    # patch to fix use of GNUCXX to instead be for both GNU
-    # and Clang compilers
-
-    patchfiles-append \
-        patch-fix-GNUCXX.devel.diff
+    set GR_VERSION_INFO "Release3.7"
 
-    # patch to fix Boost #include
-
-    patchfiles-append \
-        patch-gnuradio-runtime_lib_buffer.cc.boost_include_fix.devel.diff
-
-    # patch to fix type usage of MP package
-
-    patchfiles-append \
-        patch-fix-mp.devel.diff
-
-    # patch to fix installing CTRLPORT Python apps location
-
-    patchfiles-append \
-        patch-fix-ctrlport-python-app-install-location.devel.diff
-
-    # patch to fix FindLog2cpp.cmake script to correct cache LOG4CPP_FOUND
-
-    patchfiles-append \
-        patch-fix-FindLog4cpp.devel.diff
+    # still require cppunit for testing; NOTE: cppunit is checked for
+    # during configure, so we need it to be in depends_lib or
+    # depends_build to be used correctly. Choose the latter since it's
+    # not required for runtime; just for build/test.
+    depends_build-append \
+        port:cppunit
 
-    # patch to fix unused RPC variables
+    github.livecheck.regex  {(3.7.[0-9.]+)}
 
-    patchfiles-append \
-        patch-fix-rpc-unused-vars.devel.diff
+}
 
-    # temporary patch to fix using Thrift 0.9.2 through 0.12.0
+subport gnuradio-next {
 
-    patchfiles-append \
-        patch-fixes_for-thrift.devel.diff
+    name gnuradio-next
 
-} else {
+    github.setup gnuradio gnuradio 5099fd9a492face0facfc4a8a6923d5ce76b7b31
+    version   20191003-[string range ${github.version} 0 7]
```

Comment:
We'll want to update this; not sure if it's really necessary in this PR, but ... if possible I'd say to go for it!

Reply:
this subport track master?

Reply:
I move to that branch.